lib/src/rules/always_specify_types.dart

```diff
@@ -103,8 +120,19 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitDeclaredIdentifier(DeclaredIdentifier node) {
-    if (node.type == null) {
-      rule.reportLintForToken(node.keyword);
+    var keyword = node.keyword;
+    if (node.type == null && keyword != null) {
+      var element = node.declaredElement;
+      if (element is VariableElement) {
+        if (keyword.keyword == Keyword.VAR) {
+          rule.reportLintForToken(keyword,
+              arguments: [keyword.lexeme, element!.type],
```

Comment:
I'm surprised by the `!`.  Is that needed?  (I'd expect the `is` check to ensure a non-null value?)

Reply:
https://github.com/dart-lang/sdk/issues/50913

Reply:
Thanks!